apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -59,7 +59,7 @@
 
   <datahub-record-section
     id="resources"
-    title="Ressources et liens"
+    title="ressources.and.links"
```

Comment:
I think keys similar to the ones already present in the component (like `record.metadata.resources` for example) would be better following our current structure of translations keys.

Also, they should be present in all translation files. You can run `npm run i18n:extract` for that. Just be careful, as sometimes there have been unrelated changes in the translation files when running this script in the recent pas, I think.